Src/FluentAssertions/Primitives/StringAssertions.cs

```diff
@@ -394,7 +394,10 @@ public AndConstraint<TAssertions> NotMatch(string wildcardPattern, string becaus
         /// The regular expression with which the subject is matched.
         /// </param>
         /// <param name="occurrenceConstraint">
-        /// The excpected occurences of regex matches
+        /// A constraint specifying the expected amount of times a regex should match a string.
+        /// It can be created by invoking static methods Once, Twice, Thrice, or Times(int)
+        /// on the classes <see cref="Exactly"/>, <see cref="AtLeast"/>, <see cref="MoreThan"/>, <see cref="AtMost"/>, and <see cref="LessThan"/>.
+        /// For example, <see cref="Exactly.Times(int)"/> or <see cref="LessThan.Twice()"/>.
```

Comment:
🔁 Newlines in documentation will not be kept when rendering in the IDE. If you intend these lines as separate lines (I think that makes it more readable) surround each line with a `<para></para>` tag

Reply:
Thanks for the hint.. will add this soon

Reply:
![image](https://user-images.githubusercontent.com/44049228/160808824-4d3ee208-dd60-49ee-9ae3-b6a1daa6df9f.png)

Hmm..

Reply:
Ok, maybe not every IDE, but at least Visual Studio:
![image](https://user-images.githubusercontent.com/10671831/160809764-ba4c97a6-df66-412d-9059-81a12343c328.png)
_(I added the XML to another method, but just for a test)_

Reply:
Ok, apparently Visual Studio only applies line breaks in tooltip to the _summary_, and not the _param_, even with `<para>`.

It does apply the line breaks in the object browser, but that's quite obscure.
![image](https://user-images.githubusercontent.com/10671831/160811251-b03cf864-f0ec-4bc0-97a0-4f2d70e6317c.png)


So forget what I asked 😞  does not seem worth the trouble.